conf.yaml

```diff
@@ -71,6 +71,12 @@ targets:
     hosts:
       - bing.com
       - www.bing.com
+    twitter: "@bing"
+  CenturyLink (formerly Level3):
+    href: https://www.centurylink.com
+    hosts:
+      - www.centurylink.com
+    twitter: "@CenturyLink @CenturyLinkHelp"
```

Comment:
The code currently only supports a single twitter handle, see https://github.com/andir/ipv6.watch/blob/398b2beca4b3094037001cb2858bc59e7108ee18/templates/index.jinja2#L71-L77.

Reply:
It works flawless. I've tested it. The twitter handle is a plain string wich gets passed. It would not work if `twitter` would be a list.

You already do that with Twitch.

Reply:
Indeed, thanks for the hint.